api/lorawan.proto

```diff
@@ -240,6 +240,16 @@ message DataRate {
   };
 }
 
+// Fields that are specific for transmission on a gateway 
+message Downlink{
```

Comment:
Don't make this top-level but declare in `TxSettings`, as they are local to that